### ISSUE TITLE
[ID Card] Form control does not have a corresponding label

### ISSUE THREAD

**Issue by degraphic** [veshu/accessibility-bughunt-bluesource#84]
<h2>Steps to reproduce the behavior</h2>
1. Go to Wellmark app. <br>
2. Press Click PROFILE then Click GET ID CARD or access this link http://localhost:4200/my-plans/id-card<br> 
3. Then Click  WAVE Web Accessibility Tool from Extention On Chrome

<h2>Actual behavior</h2>
We can see Form control does not have a corresponding label inside the form

<h2>Expected behavior</h2>
I think we can add Label inside the form to fix the error

**Device**: Laptop MacbookPro
**Operating System**: MacOs Mojave 10.14.6
**Browser(with version)**: Chrome Version 78.0.3904.70 (Official Build) (64-bit)

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non text content
**Tools used**: WAVE

**Screenshoot:**
<br>

![Screen Shot 2019-11-01 at 01 31 38](https://user-images.githubusercontent.com/4364214/67976027-d94b9e80-fc47-11e9-926c-d2d45a6173cb.png)
<br>
![Screen Shot 2019-11-01 at 01 30 32](https://user-images.githubusercontent.com/4364214/67976029-d9e43500-fc47-11e9-927d-3380dddd2e41.png)

**Comment by drasticdpk** [comment]
WCAG Category: 1. Perceivable

![image](https://user-images.githubusercontent.com/25297859/68069907-05256c00-fd8f-11e9-9a9b-87e9d905ac8d.png)
